web-server/v0.4/components/Layout/IterationSummary.js

```diff
@@ -43,16 +40,17 @@ class IterationSummary extends React.Component {
     document.body.appendChild(script4);
   }
 
-  iframe() {
+  iframe(controllerName, resultName, iterationNumber, iterationName, closestSample) {
     return {
-      __html: '<iframe src="http://pbench.perf.lab.eng.bos.redhat.com/results/' + window.location.href.split('/dashboard/results/').pop() + '/uperf.html" width="100%" height="2000"></iframe>'
+      __html: '<iframe style="overflow: hidden; overflow-y: hidden; height: 650px; border: none; margin: 0; padding: 0; scrolling: none" src="http://pbench.perf.lab.eng.bos.redhat.com/results/' + controllerName + '/' + resultName + '/' + iterationNumber + '-' + iterationName + '/' + 'sample' + closestSample + '/uperf.html" width="100%" height="1000"></iframe>'
```

Comment:
Can we find a way to remove this hard-coded `http://pbench.perf.lab.eng.bos.redhat.com` URL prefix?